### PR TITLE
[10.0][ADD] EDI code and operator for partners

### PR DESCRIPTION
**Pull request by jarmokortetjarvi**
Adds an EDI code field and operator for partners

### CODE REVIEW

File: l10n_fi_edicode/models/res_partner_operator_einvoice.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# -*- coding: utf-8 -*-
+from odoo import api, fields, models
+
+
+class ResPartnerOperatorEinvoice(models.Model):
+    _name = 'res.partner.operator.einvoice'
+
+    name = fields.Char(
+        string='Operator',
```
Comment (mlaitinen) [inline l10n_fi_edicode/models/res_partner_operator_einvoice.py:9]: Make required?

File: l10n_fi_edicode/models/res_partner_operator_einvoice.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    )
+    identifier = fields.Char(
+        string='Identifier',
```
Comment (mlaitinen) [inline l10n_fi_edicode/models/res_partner_operator_einvoice.py]: Should it be required too?

File: l10n_fi_edicode/data/res_partner_operator_einvoice.xml [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+        <field name="name">DNB Bank ASA</field>
+        <field name="identifier">DNBAFIHX</field>
+    </record>
+
+    <record id="operator_einvoice_enfo" model="res.partner.operator.einvoice">
+        <field name="name">Enfo Zender Oy</field>
+        <field name="identifier">003714377140</field>
+    </record>
+
+    <record id="operator_einvoice_handelsbanken" model="res.partner.operator.einvoice">
```
Comment (mlaitinen) [inline l10n_fi_edicode/data/res_partner_operator_einvoice.xml]: Handelsbanken twice?

File: l10n_fi_edicode/data/res_partner_operator_einvoice.xml [hunk 4]
```diff
@@ -0,0 +1,133 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
```
Comment (mlaitinen) [inline l10n_fi_edicode/data/res_partner_operator_einvoice.xml:2]: According to http://verkkolasku.tieke.fi there are 28 operators currently and only 25 are listed. 

- Lexmark is missing
- Netbox is missing
- Palette Software is missing
- Enfo Zender Oy is now Ropo Capital Oy
- TeliaSonera is now Telia Finland Oyj.

**Review by mlaitinen — CHANGES_REQUESTED** [review]
Some changes required but otherwise looks good

**Review by mlaitinen — APPROVED** [review]
Nice :+1:

**Comment by jarmokortetjarvi** [comment]
Changing the ids will break reference (ref) backwards-compatibility. The worst case scenario (the module is already installed) should be that there will be duplicate operators, and existing partner-operator linking will point to incorrect operator.

**Review by mlaitinen — CHANGES_REQUESTED** [review]
The model res.partner.operator.einvoice has no access rules, consider adding one. E.g. access_res_partner_operator_einvoice,access_res_partner_operator_einvoice,model_res_partner_operator_einvoice,,1,0,0,0

**Comment by mlaitinen** [comment]
Umm.. what's wrong with runbot? Can't see any errors in the logs.

**Review by mlaitinen — APPROVED** [review]

**Comment by mlaitinen** [comment]
I approved this PR. Where could we find someone else to approve it? @pedrobaeza ?

**Review by aisopuro — APPROVED** [review]
LGTM

**Pull request merged**